stdlib/public/core/VarArgs.swift

```diff
@@ -467,6 +491,138 @@ final internal class _VaListBuilder {
   internal var storage: ContiguousArray<Int>
 }
 
+#elseif arch(arm64) && !(os(macOS) || os(iOS) || os(tvOS) || os(watchOS) || os(Windows))
+
+@_fixed_layout // FIXME(sil-serialize-all)
+@_versioned // FIXME(sil-serialize-all)
+final internal class _VaListBuilder {
```

Comment:
This seems heavily duplicated with the x86_64 / s390x implementation. Is it possible to merge them?